charts/stable/speedtest-exporter/templates/prometheusrules.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if and .Values.metrics.enabled .Values.metrics.prometheusRule.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+    {{- with .Values.metrics.prometheusRule.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  groups:
+    - name: {{ include "common.names.fullname" . }}
+      rules:
+        - alert: SpeedtestExporterAbsent
+          annotations:
+            description: Speedtest Exporter has disappeared from Prometheus target discovery.
+            summary: Speedtest Exporter is down.
+          expr: |
+            absent(up{job="{{ include "common.names.fullname" . }}"} == 1)
+          for: {{ .Values.metrics.serviceMonitor.interval }}
+          labels:
+            severity: critical
+        - alert: SpeedtestSlowInternetDownload
+          annotations:
+            description: Internet download speed is averaging {{ "{{ humanize $value }}" }} Mbps.
+            summary: SpeedTest slow internet download.
+          expr: |
+            avg_over_time(speedtest_download[4h])
+              < {{ .Values.metrics.prometheusRule.downloadLimit }}
+          for: 0m
+          labels:
+            severity: warning
+        - alert: SpeedtestSlowInternetUpload
+          annotations:
+            description: Internet upload speed is averaging {{ "{{ humanize $value }}" }} Mbps.
+            summary: SpeedTest slow internet upload.
+          expr: |
+            avg_over_time(speedtest_upload[4h])
+              < {{ .Values.metrics.prometheusRule.uploadLimit }}
+          for: 0m
+          labels:
+            severity: warning
+        - alert: SpeedtestHighPingLatency
+          annotations:
+            description: Internet ping latency is averaging {{ "{{ humanize $value }}" }} ms.
+            summary: SpeedTest high ping latency.
+          expr: |
+            avg_over_time(speedtest_ping_latency_milliseconds[2h])
```

Comment:
```suggestion
            avg_over_time(speedtest_ping_latency_milliseconds{job=~".*{{ include "common.names.fullname" . }}-metrics.*"}[2h])
```

Reply:
I think we can drop the `-metrics` and just let regex find it. Just in case someone manages to change the metrics port name.